app_dart/lib/src/request_handlers/reset_prod_task.dart

```diff
@@ -45,6 +49,8 @@ class ResetProdTask extends ApiRequestHandler<Body> {
     final String encodedKey = requestData[taskKeyParam] as String;
     final ClientContext clientContext = authContext.clientContext;
     final KeyHelper keyHelper = KeyHelper(applicationContext: clientContext.applicationContext);
+    final String owner = requestData[ownerParam] as String ?? 'flutter';
```

Comment:
Have you thought about moving these to query params? If this was a GET request. we could use the CacheRequestHandler to rate limit this handler with specific args

Reply:
The reason it was added as post is because per convention rest apis modifying state are usually post and get requests do not modify state.